main.js

```diff
@@ -148,7 +148,7 @@ module.exports = function(options) {
 				directory + '/views/partials'
 			],
 			defaultLayout: false,
-			layoutsDir: options.layoutsDir || __dirname + '/layouts',
+			layoutsDir: options.layoutsDir || process.cwd() + '/client/layouts',
```

Comment:
We've built APIs on top of this that shouldn't have any client-side deps though?